packages/serve/src/types.ts

```diff
@@ -1,9 +1,10 @@
 export type devServerOptionsType = {
-    bonjour?: boolean;
+    bonjour?: boolean | Record<string, any>;
```

Comment:
`any` for record type?

Reply:
yep they are from external package so left out as any - https://www.npmjs.com/package/bonjour#publishing